x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/action/EqlSearchResponse.java

```diff
@@ -71,6 +70,7 @@
  *         EqlSearchResponse.Hits hits = new EqlSearchResponse.Hits(null, null, counts, totals);
  *         EqlSearchResponse response = new EqlSearchResponse(hits, 5, false);
  */
+
```

Comment:
Can you remove this, please?

Reply:
Sure. It was initially useful to have while we were iterating on the response shape.